resources/lib/KodiHelper.py

```diff
@@ -345,6 +361,7 @@ def build_profiles_listing (self, profiles, action, build_url):
             xbmcplugin.addDirectoryItem(handle=self.plugin_handle, url=url, listitem=li, isFolder=True)
             xbmcplugin.addSortMethod(handle=self.plugin_handle, sortMethod=xbmcplugin.SORT_METHOD_LABEL)
         xbmcplugin.endOfDirectory(self.plugin_handle)
+        self.SetView('login')
```

Comment:
can you be more clear?

Reply:
Sure.

Declare
```
VIEW_LOGIN = 'login'
VIEW_FOLDER = 'folder'
VIEW_SOMTHING...
```

At the top of the file & use the constants in the actual methods instead of the strings:

```
self.SetView('login')
```

should become

```
self.set_custom_view(VIEW_LOGIN)
```

```
self.SetView('folder')
```

should become

```
self.set_custom_view(VIEW_FOLDER)
```

ettc.

Reply:
It is changed with last commit - also a refresh fix is included to make sure view is changed. Hope it's ok for you now.